gnssanalysis/gn_io/sp3.py

```diff
@@ -44,9 +41,7 @@
 SP3_POS_STD_NODATA = -100
 
 
-def sp3_pos_nodata_to_nan(
-    sp3_df: _pd.DataFrame
-) -> None:
+def sp3_pos_nodata_to_nan(sp3_df: _pd.DataFrame) -> None:
     """
     Converts the SP3 Positional column's nodata values (0.000000) to NaNs.
     See https://files.igs.org/pub/data/format/sp3_docu.txt
```

Comment:
Just on the path of filling in docstrings for older functions, it might be a good idea that for all functions that we touch in future PRs, we should aim to complete the one-line sphinx docstring.
We can either start with your PR here, or for all future ones